models/classes/LtiProvider/LtiProviderFactory.php

```diff
@@ -71,21 +70,27 @@ public function createFromResource(core_kernel_classes_Resource $resource): LtiP
 
     public function createFromArray(array $provider): LtiProvider
     {
-        $keys = ['uri', 'label', 'key', 'secret', 'callback_url'];
+        $ltiVersion = $provider[ConfigurableLtiProviderRepository::LTI_VERSION] ?: '1.1';
```

Comment:
```suggestion
        $ltiVersion = $provider[ConfigurableLtiProviderRepository::LTI_VERSION] ?? '1.1';
```
Otherwise throws an error if index does not exist.